kinto_algolia/indexer.py

```diff
@@ -74,19 +73,19 @@ def search(self, bucket_id, collection_id, **kwargs):
         indexname = self.indexname(bucket_id, collection_id)
         index = self.client.init_index(indexname)
         query = kwargs.pop("query", "")
-        return index.search(query, args=kwargs)
+        return index.search(query, kwargs)
 
     def flush(self):
-        response = self.client.list_indexes()
+        response = self.client.list_indices()
         for index in response["items"]:
             indexname = index["name"]
             if indexname.startswith(self.prefix):
                 index = self.client.init_index(indexname)
-                res = index.clear_index()
-                self.tasks.append((indexname, res["taskID"]))
-                res = self.client.delete_index(indexname)
-                self.tasks.append((indexname, res["taskID"]))
-        self.join()
+                index.clear_objects().wait()
+                index.delete().wait()
+
+    def isalive(self):
+        self.client._transporter.read(Verb.GET, "1/isalive", {}, None)
```

Comment:
That's eventually how I did it, but I guess it would have been better to keep the `isalive()` method.

Reply:
@Natim Sounds good at the moment, as this moment is not in our plans for the v2.

Reply:
Why did you choose to remove it?